src/Components/Results/Header/Header.tsx

```diff
@@ -15,8 +18,34 @@ const Buttons = () => {
 
 const ProgramsHeader = () => {
   const { programs } = useResultsContext();
+  const { theme } = useContext(Context);
+  const taxCredit = calculateTotalValue(programs, 'Tax Credit');
 
-  return <div>{programs.length}</div>;
+  const estimatedMonthlySavings = programs.reduce(
+    (eachEstimatedMonthlySavings, program) => eachEstimatedMonthlySavings + program.estimated_value,
+    0,
+  );
+
+  return (
+    <CardContent sx={{ backgroundColor: theme.secondaryBackgroundColor }}>
+      <div className="results-header">
+        <div className="results-header-programs-count-text">
+          <div className="results-header-programs-count">{programs.length}</div>
+          <div>Programs Found</div>
+        </div>
+        <div style={{ marginLeft: '15px' }}>
+          <div className="results-header-values">
+            ${Math.round(estimatedMonthlySavings / 12).toLocaleString()} <div className="info-circle">i</div>
```

Comment:
We can remove the help icons for now because that feature is not implemented yet.
```suggestion
            ${Math.round(estimatedMonthlySavings / 12).toLocaleString()}
```